frontend/src/components/CreateCollectionModal/components/Content/index.tsx

```diff
@@ -38,26 +47,51 @@ enum FIELD_NAMES {
   CONTACT_EMAIL = "contact-email",
 }
 
+const AddLinkButton = () => (
+  <Button outlined intent={Intent.PRIMARY}>
+    Add Link
+  </Button>
+);
+
 const Content: FC<Props> = (props) => {
-  const [isValid, setIsValid] = useState(false);
+  const isEditCollection = !!props.id;
+  const initialBooleanState = isEditCollection;
```

Comment:
If we're editing a collection we want a lot of boolean logic to be initialized as `true`

Reply:
SGTM!